packages/opentelemetry-plugin-mysql/src/mysql.ts

```diff
@@ -179,11 +179,13 @@ export class MysqlPlugin extends BasePlugin<typeof mysqlTypes> {
       if (arguments[1]) {
         // this is the callback passed into a query
         // no need to unwrap
-        shimmer.wrap(
-          arguments[1],
-          'query',
-          thisPlugin._patchQuery(arguments[1])
-        );
+        if (!arguments[1].query.__wrapped) {
```

Comment:
Optional: `core` module contains `isWrapped` function, maybe we can use that here?